evalml/pipelines/components/transformers/preprocessing/polynomial_detrender.py

```diff
@@ -56,8 +51,7 @@ def fit(self, X, y=None):
         if y is None:
             raise ValueError("y cannot be None for PolynomialDetrender!")
         y_dt = infer_feature_types(y)
-        y_series = _convert_woodwork_types_wrapper(y_dt.to_series())
-        self._component_obj.fit(y_series)
+        self._component_obj.fit(y_dt)
```

Comment:
Super nit-pick (and not really added by your PR) but why is it called y_dt? Is dt in place for data table? :o If so, maybe we can rename as y_ww or something?

Reply:
Yea I'll make that change!